src/jaxsim/api/contact.py

```diff
@@ -167,11 +167,11 @@ def estimate_model_height(model: Model.JaxSimModel) -> jtp.Float:
             model=model, soft_contacts_params=soft_contacts.SoftContactsParams()
         )
 
-        W_p_CoM = Model.com_position(model=model, data=zero_data)
+        W_p_CoM = Model.com_position(model=model, data=zero_data)[2]
```

Comment:
```suggestion
        W_pz_CoM = Model.com_position(model=model, data=zero_data)[2]
```

Reply:
Thanks, done in https://github.com/ami-iit/jaxsim/pull/98/commits/d378d565d77d6d156ed16bbc8499444478d3fa95